cpp/src/barretenberg/stdlib/hash/sha256/sha256.test.cpp

```diff
@@ -148,43 +146,43 @@ TEST(stdlib_sha256, test_duplicate_proving_key)
 // TEST(stdlib_sha256_plookup, test_round)
```

Comment:
Should this be removed? Do we know why its commented out?

Reply:
I don't, so I can't remove it. 3 years, though